src/components/overlays/Modal/Popover.tsx

```diff
@@ -3,11 +3,11 @@ import { Overlay } from './Overlay';
 import { forwardRef, HTMLAttributes } from 'react';
 import { useModal, useOverlay } from '@react-aria/overlays';
 import { Base } from '../../Base';
-import { useContextStyles } from '../../../providers/StylesProvider';
+import { useContextStyles } from '../../../tasty/providers/StylesProvider';
```

Comment:
I think we should add these imports to the top-level index.ts file under /tasty. Making so we declare that these imports are part of tasty public contract as well.

Reply:
Definitely